project1/css/index.css

```diff
@@ -62,3 +62,198 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+/* start of code: Challenge 1
+
+link {
+	right: 0;
+}
+
+.logo {
+	display: inline-block;
+	align-content: right;
+}
+li{
+	display: inline-block;
+}
+a {
+	color: grey;
+	text-decoration: none;
+	display: inline-block;
+	margin: 50px 50px 50px 80px;
+}
+
+.cta-text{
+	margin-top: 100px;
+	font-size: 50px;
+	display: inline-block; 
+}
+
+.cta .img{
+	display: inline-block;
+}
+
+.contact{
+	margin: 40px;
+}
+
+.line {
+	border: 1px grey solid;
+	margin: 45px;
+}
+
+.spaces{
+	border: 5px white solid;
+}
+
+.top-content{
+	margin: 20px;
+	display: inline-block;
+}
+
+.main-content .top-content .text-content p {
+	display: inline-block;
+	height: 100px;
+	width: 400px;
+}
+
+.middle{
+	display: block;
+	margin: auto;
+	text-align: center;
+}
+
+.bottom-content{
+	margin: 40px;
+}
+.main-content .bottom-content .text-content p{
+	display: inline-block;
+	height: 100px;
+	width: 400px;
+}
+
+footer{
+	margin-top: 5px;
+	margin-left: 40px;
+} */
+
+
+
+
+
+/* Challenge 2 */
+
+.container {
```

Comment:
A typical standard is to keep all of your CSS rules tabbed in one time. An image example from my editor: 
![image](https://user-images.githubusercontent.com/24354711/37108034-cbd2e0a8-21ea-11e8-8128-e143dfc46296.png)

You can see that it's all one tab in for the selector, then one tab further for the rules within it. It's okay if you don't want to conform to that and you like it this way, as long as you keep it uniform, but I figured you may want to know what is considered "standard"